WinPort/src/Backend/TTY/TTYInputSequenceParser.cpp

```diff
@@ -386,10 +650,19 @@ size_t TTYInputSequenceParser::ParseEscapeSequence(const char *s, size_t l)
 		return 5;
 	}
 
-	size_t r = ParseNChars2Key(s, l);
+	size_t r = 0;
+
```

Comment:
why separate line?

Reply:
not needed any more, fixed. in my first experiments kittys seqs were parsed first, but it makes kitty parser intercept some traditionals seqs and that is not what we need probably. also kittys parser falls back to classical parser for parsing SS3 format.